xwiki-platform-core/xwiki-platform-eventstream/xwiki-platform-eventstream-api/src/main/java/org/xwiki/eventstream/events/EventStreamAddedEvent.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.eventstream.events;
+
+import org.xwiki.eventstream.Event;
+
+/**
+ * Event triggered when a new event is registered inside of the {@link org.xwiki.eventstream.EventStream}. This event
+ * contains a reference to the actual event that has been registered in the event stream.
+ *
+ * @since 9.6RC1
+ * @version $Id$
+ */
+public class EventStreamAddedEvent extends AbstractEventStreamEvent
+{
+    /**
+     * Constructs a new {@link EventStreamAddedEvent}.
+     */
+    public EventStreamAddedEvent()
```

Comment:
See https://github.com/aubincleme/xwiki-platform/blob/xwiki-14353/xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/email/LiveNotificationEmailListener.java#L106 for example